pyprocessing/math.py

```diff
@@ -183,14 +185,14 @@ def __init__(self, *args, x=0, y=0, z=None):
             self.seq = [x, y, z] if z is not None else [x, y]
 
     def __copy__(self):
-        return PVector(*self)
+        return self.copy()
 
     def __add__(self, other):
         if isinstance(other, complex):
             other = (other.real, other.imag)
 
         if hasattr(other, '__iter__'):
```

Comment:
Forgot to mention it, use isinstance instead (`isinstance(other, collections.abc.Iterable)` for instance)